packages/flutter_tools/lib/src/commands/build_bundle.dart

```diff
@@ -108,6 +109,7 @@ class BuildBundleCommand extends BuildSubCommand {
         }
         break;
       case TargetPlatform.windows_x64:
+      case TargetPlatform.windows_arm64:
```

Comment:
Looks like this test can be updated to include ARM https://github.com/flutter/flutter/blob/35afe1bdac653c752c5794cf877b64238a4c9e4c/packages/flutter_tools/test/commands.shard/permeable/build_bundle_test.dart#L87